example-locale/src/components/layout.tsx

```diff
@@ -0,0 +1,45 @@
+import { Outlet, Link } from "react-router-dom"
+import appwriteLogo from "../../public/appwrite.svg"
```

Comment:
This should probably be moved too, right?

Reply:
i think we are okay with this one.